packages/mdc-list/README.md

```diff
@@ -1,49 +1,23 @@
-<!--docs:
-title: "Lists"
-layout: detail
-section: components
-excerpt: "Lists present multiple line items vertically as a single continuous element."
-iconId: list
-path: /catalog/lists/
--->
-
 # Lists
 
-<!--<div class="article__asset">
-  <a class="article__asset-link"
-     href="https://material-components-web.appspot.com/list.html">
-    <img src="{{ site.rootpath }}/images/mdc_web_screenshots/lists.png" width="365" alt="Lists screenshot">
-  </a>
-</div>-->
-
-MDC List provides styles which implement [Material Design Lists](https://material.io/guidelines/components/lists.html) -
-"A single continuous column of tessellated subdivisions of equal width." Both single-line and two-line lists are
-supported (with three-line lists [planned](https://github.com/material-components/material-components-web/issues/31)).
-MDC Lists are designed to be accessible and RTL aware.
+MDC List provides styles which implement Material Design Lists - "A single continuous column of tessellated subdivisions of equal width." Both single-line and two-line lists are supported (with three-line lists planned). MDC Lists are designed to be accessible and RTL aware.
 
 ## Design & API Documentation
 
-<ul class="icon-list">
-  <li class="icon-list-item icon-list-item--spec">
-    <a href="https://material.io/guidelines/components/lists.html">Material Design guidelines: Lists</a>
-  </li>
-  <li class="icon-list-item icon-list-item--link">
-    <a href="https://material-components-web.appspot.com/list.html">Demo</a>
-  </li>
-</ul>
+* [Material Design guidelines: List](https://material.io/guidelines/components/lists.html)
+* [Demo](https://material-components-web.appspot.com/list.html)
 
 ## Installation
-
 ```
 npm install --save @material/list
 ```
 
 ## Usage
 
-### Basic Lists
 
-A basic list consists simply of the list itself, and list items taking up one line.
+## Single-Line List
```

Comment:
I think for the Usage section, we should separate the sections first by lanaguage (HTML, then CSS/Sass) and then by type of list (single-line, two-line, etc). Then we can consolidate the CSS/Sass into a single reference table, instead having a CSS table for every type of list. E.g.
```
### HTML Structure
#### Single-line list
...
#### Two-line list
...
#### List groups
..
#### List dividers
...
### CSS Classes
...
### Sass Mixins
...
```
See the [Text Field README](https://github.com/material-components/material-components-web/blob/master/packages/mdc-textfield/README.md) for an example of this.

The reason for this is because the first thing developers want to know is what types of lists we provide and the HTML structure that they can copy/paste to use it, which is why we should put all of that at the beginning. Then, if they're interested in understanding the structure or CSS behind it, they can look at the CSS/Sass reference tables. There's also a lot of overlap in the CSS tables (`mdc-list-item` appears in multiple tables, for example) so we can consolidate all the tables into one, to be as concise as possible.

But that's my thinking, what are your thoughts on this?

Reply:
Also maybe we should add a one-line description for all the different types of lists (single-line, two-line, etc).

Reply:
Yas, definitely. That makes sense to me. As someone new to MDC I would want all the classes in one place. I think the dev will be smart enough to figure out where the class should go.

And yes I'll add a one-liner for each type of list